### PR TITLE
Update 6.5.4

### PR DESCRIPTION
**Pull request by Jrice1317**
- Updated version and hash
- Alphabetized run section
- Updated pinnings
- Added install_reqs
- Updated urls
- Added license and doc_source urls

Jira: https://anaconda.atlassian.net/browse/PKG-155
https://github.com/jupyter/nbconvert/tree/main

### CODE REVIEW

**Review by varlackc — APPROVED** [review]
LGTM

**Review by ELundby45 — APPROVED** [review]
LGTM. I'll just add a reminder to remove the abs.yaml before deploying.

File: abs.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+channels:
```
Comment (katietz) [inline abs.yaml]: please remove this channel stuff

File: recipe/meta.yaml [hunk 2]
```diff
@@ -64,17 +67,19 @@ test:
     - jupyter nbconvert nbconvert/tests/files/notebook1.ipynb --to html
 
 about:
-  home: https://jupyter.org
+  home: https://nbconvert.readthedocs.io/en/latest/
   license: BSD-3-Clause
   license_file: LICENSE
   license_family: BSD
+  license_url: https://github.com/jupyter/nbconvert/blob/6.5.3/LICENSE
```
Comment (katietz) [inline recipe/meta.yaml]: there is a license_file in package.  No need for the license_url.  It is superflous, even problematic mid/long term

File: recipe/meta.yaml [hunk 3]
```diff
@@ -64,17 +67,19 @@ test:
...
   summary: Converting Jupyter Notebooks
   description: |
     The nbconvert tool, jupyter nbconvert, converts notebooks to various other
     formats via Jinja templates. The nbconvert tool allows you to convert an
     .ipynb notebook file into various static formats.
-  doc_url: https://nbconvert.readthedocs.org/
+  doc_url: https://nbconvert.readthedocs.io/en/latest/
   dev_url: https://github.com/jupyter/nbconvert
+  doc_source_url: https://github.com/jupyter/nbconvert/tree/6.5.3/docs
```
Comment (katietz) [inline recipe/meta.yaml]: doc_source_url is superflous.  there is already a doc_url

**Review by katietz — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 3 above)
Comment (katietz) [reply recipe/meta.yaml]: done it for you.  Next time please adjust before asking me for a second review

File: recipe/meta.yaml (hunk 2 above)
Comment (katietz) [reply recipe/meta.yaml]: done for you ...

File: abs.yaml (hunk 1 above)
Comment (katietz) [reply abs.yaml]: done for you

**Comment by katietz** [comment]
please upload first tinycss2 before building this package!

**Comment by katietz** [comment]
after uploading its dependencies, it can be build without private channel information abs.yaml, as you see now

**Review by katietz — APPROVED** [review]
after all issues are resolved, final build was successful without private channel, the PR is ready for approval. and it looks good to me now.

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**